pkg/apis/componentconfig/types.go

```diff
@@ -610,6 +610,16 @@ type KubeControllerManagerConfiguration struct {
 	// concurrentGCSyncs is the number of garbage collector workers that are
 	// allowed to sync concurrently.
 	ConcurrentGCSyncs int32 `json:"concurrentGCSyncs"`
+	// nodeEvictionRate is the number of Node evictions NodeController does in ordinary circumstances
+	NodeEvictionRate float32 `json:"nodeEvictionRate"`
+	// secondaryNodeEvictionRate is the number of Node evictions NodeController does when zone is unhealty
+	SecondaryNodeEvictionRate float32 `json:"secondaryNodeEvictionRate"`
+	// largeClusterSizeThreshold is the number of Nodes from which NodeController treats a cluster as "large"
+	// for eviction logic purposes
+	LargeClusterSizeThreshold int32 `json:"largeClusterSizeThreshold"`
+	// unhealthyZoneThreshold is a percentage of Nodes that needs to be not Ready for NodeController to treat a zone
```

Comment:
Would it be better to make this be a fraction rather than percentage? (e.g. 0.33 instead of 33)